File: C# (ASP.NET & WPF)/C# - INSY7315 - EliteRentals - (Web App, Mobile App & API)/EliteRentals-WebApp/Models/DTOs/PaymentDto.cs
namespace EliteRentals.Models.DTOs
{
    public class PaymentDto
    {
        public int PaymentId { get; set; }
        public int TenantId { get; set; }
        public string? TenantName { get; set; } // Derived from tenant
        public decimal Amount { get; set; }
        public string Status { get; set; } = "Pending";
        public DateTime Date { get; set; }
        public string? ProofType { get; set; }
        public byte[]? ProofData { get; set; }
    }
}

File: C# (ASP.NET & WPF)/C# - INSY7315 - EliteRentals - (Web App, Mobile App & API)/Elite-Rentals-Mobile-App/app/src/main/java/com/rentals/eliterentals/PaymentDto.kt
package com.rentals.eliterentals

data class PaymentDto(
    val paymentId: Int,
    val tenantId: Int,
    val amount: Double,
    val method: String?,
    val date: String?,
    val status: String,
    val proofType: String?,
)

data class PaymentStatusDto(
    val status: String
)

